### PR TITLE
rewrite restricted quantifiers # 4

### PR DESCRIPTION
**Pull request by wlammen**
finishes all up-to-ax5 only theorems.

### CODE REVIEW

**Comment by avekens** [comment]
I would like to propose to use subsubsections for each (closely related group of) definition. First of all to help to better understand the process of rearrangements in progress, but also to have a clean separation of the different concepts. The subsubsections of subsection " 2.1.5  Restricted quantification" could be named as follows:

* 2.1.5.1 Restricted universal and existential quantification 
* 2.1.5.2 Restricted existential uniqueness and "at most one"
* 2.1.5.3 Restricted class abstraction

By the way, we have already separate (sub)subsections for unrestricted quantification:
* 1.4.1.1  Existential quantifier
* 1.6.1  Uniqueness: the at-most-one quantifier
* 1.6.2  Unique existence: the unique existential quantifier 
* 2.1.2.1  Class abstractions

**Comment by wlammen** [comment]
> 
    2.1.5.1 Restricted universal and existential quantification
    2.1.5.2 Restricted existential uniqueness and "at most one"
    2.1.5.3 Restricted class abstraction

We can do that.

File: set.mm [hunk 1]
```diff
@@ -29488,41 +29488,238 @@ atomic formula (class version of ~ elsb1 ).  Usage of this theorem is
...
+
+  ${
+    $d x y ph $.  $d x y ps $.  $d y A $.
+    ralrimdvva.1 $e |- ( ( ph /\ ( x e. A /\ y e. B ) ) -> ( ps -> ch ) ) $.
+    $( Inference from Theorem 19.21 of [Margaris] p. 90.  (Restricted
+       quantifier version with double quantification.)  (Contributed by NM,
+       2-Feb-2008.) $)
+    ralrimdvva $p |- ( ph -> ( ps -> A. x e. A A. y e. B ch ) ) $=
+      ( cv wcel wa wi ex com23 ralrimdvv ) ABCDEFGADIFJEIGJKZBCAPBCLHMNO $.
+  $}
```
Comment (icecream17) [inline set.mm:29508]: ralrimdvva et al seems 2 dimensional while rgen3 above is 3, unless I'm misunderstanding dimensions, or there is no preference for second vs third dimension

similar with 3reeanv and 2ralor

**Review by icecream17 — APPROVED** [review]

File: set.mm (hunk 1 above)
Comment (wlammen) [reply set.mm:29508]: So far, I only distinguish between one- and multi-dimensional, keeping theorems similar in structure like rgen2 and rgen3 still close together.  If possible, lets first establish an overall rewrite, before getting into details.  This fine tuning need not be done by me, by the way.

**Pull request merged**